src/components/common/HeadlessPopover.vue

```diff
@@ -186,40 +187,20 @@ onMounted(() => {
   if (document) {
     // handle various click events to determine how to handle the click event in a generic clickHandler function
     // we don't set any other click event listeners on purpose to avoid conflict of event listeners
-    document?.addEventListener('click', clickHandler)
+    useEventListener(document, 'click', clickHandler)
 
     if (popoverTrigger.value && props.openOnMouseover) {
-      popoverTrigger.value.addEventListener('mouseenter', showPopover)
-      popoverTrigger.value.addEventListener('focus', showPopover)
-      popoverTrigger.value.addEventListener('mouseleave', hidePopover)
-      popoverTrigger.value.addEventListener('blur', hidePopover)
+      useEventListener(popoverTrigger.value, 'mouseenter', showPopover)
+      useEventListener(popoverTrigger.value, 'focus', showPopover)
+      useEventListener(popoverTrigger.value, 'mouseleave', hidePopover)
+      useEventListener(popoverTrigger.value, 'blur', hidePopover)
```

Comment:
I think we can pass multiple events as an array to `useEventListener`, like this `useEventListener(popoverTrigger.value, ['mouseenter', 'focus'], showPopover)`.

Can you try it out and verify?